src/SCRIPTS/BF/HORUS/vtx.lua

```diff
@@ -45,19 +49,150 @@ return {
         { 5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917 }, -- RaceBand
     },
     postLoad = function (self)
-        if self.values[2] == 0 or self.values[3] == 0 or self.values[4] == 0 then
+        if (self.values[2] or 0) < 0 or (self.values[3] or 0) == 0 or (self.values[4] or 0) == 0 then
             self.values = {}
+        else
+            prevBandVal = 0                      -- reset value trackers
+            prevChanVal = 0
+            prevFreqVal = 0
+            local rFreq
+            if (self.values[7] or 0) > 0 then
+                rFreq = math.floor(self.values[6] + (self.values[7] * 256))
+            else
+                rFreq = 0
+            end
+            if (self.values[2] or 0) > 0 then    -- band != 0
+                if self.values[3] and self.values[3] > 0 then
+                    if rFreq > 0 then             
+                        prevFreqVal = rFreq
+                        prevBandVal = self.values[2]
+                        prevChanVal = self.values[3]
+                        self.fields[1].min = 0   -- make sure 'U' band allowed
+                        eepromWrite = true
```

Comment:
Scope should also be addressed on eepromWrite.  As a member, it should be accessed using self.

Reply:
Does this provide a measurable benefit?  Seems like the write is fairly benign on older versions (even though the action isn't actually saving anything new).